src/ray/util/size_literals.h

```diff
@@ -0,0 +1,99 @@
+// Copyright 2024 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// These user-defined literals makes sizes.
+
+#pragma once
+
+#include <cstdint>
+
+constexpr unsigned long long operator""_PiB(unsigned long long sz) {
+  return sz * 1024ULL * 1024ULL * 1024ULL * 1024ULL * 1024ULL;
+}
+constexpr unsigned long long operator""_PB(unsigned long long sz) {
```

Comment:
why do we use long long instead of size_t

Reply:
I don't see much difference? 
- `unsigned long long` is the largest unsigned range we could have in C++ natively
- we won't have negative value here

Reply:
ofc they are the same: size_t is just typedef of unsigned long long on 64bit system. However it has a clearer semantics: it represents a size, which is what all these "number of bytes" about.

Reply:
No strong preference, updated.

Reply:
OK, I found `size_t` doesn't compile
```
./src/ray/util/size_literals.h:22:18: error: 'constexpr size_t operator""_B(size_t)' has invalid argument list
   22 | constexpr size_t operator""_B(size_t sz) { return sz; }
      |                  ^~~~~~~~
```

Working version:
```
constexpr unsigned long long operator""_B(unsigned long long sz) { return sz; }
```
Non-workable version:
```
constexpr size_t operator""_B(size_t sz) { return sz; }
```